ethereum_history_api/oracles/src/util/abi.test.ts

```diff
@@ -12,7 +12,8 @@ describe('abi', () => {
       parameters: [publicParam, privateParam],
       param_witnesses: {},
       return_type: null,
-      return_witnesses: []
+      return_witnesses: [],
+      error_types: {}
```

Comment:
In the new version of noir_abi lib error_types field was added to Abi type, so to resolve an error I added it here